docs/software/devtools/linaro/index.md

```diff
@@ -3,27 +3,27 @@
 
 [Linaro Forge](https://docs.linaroforge.com/latest/html/forge/index.html) is a suite of profiling and debugging tools, that includes the DDT debugger and the MAP profiler.
 
-!!! note
+!!! note "Linaro DDT debugger and Linaro MAP profiler"
     We have separate user guides for the tools provided by the `linaro-forge` uenv.
     The documentation here shows how to download the uenv, and how to set up your environment.
 
     Once you are set up, follow the specific guides:
 
-    * the [DDT debugger][ref-devtools-ddt]
-    * the [MAP profiler][ref-devtools-map].
+    * [DDT debugger][ref-devtools-ddt],
+    * [MAP profiler][ref-devtools-map].
 
 ## Quickstart guide
 
-The Linaro uenv is named `linaro-forge`, and the available versions can be determined using the `uenv image find` command:
+The Linaro [uenv][ref-uenv] is named `linaro-forge`, and the available versions can be determined using the `uenv image find` command, as explained in the [uenv documentation][ref-uenv].
 
-!!! example "finding available linaro-forge versions"
+??? example "Finding available `linaro-forge` versions"
 
-    ```
-    > uenv image find linaro-forge
+    ```console
+    $ uenv image find linaro-forge
```

Comment:
I have been using `>` as the general prompt as a rule.
It is also acceptable to use `$` (maybe slightly more correct as the standard prompt for normal user).

Can this be left as a matter of choice for the author of the docs on a page, or should we pick one and use it everywhere?

Reply:
You've already seen #74 and perhaps https://github.com/eth-cscs/cscs-docs/pull/72#discussion_r2024596071. I meant to make it a formal policy, since `>` doesn't highlight correctly, but I didn't take the last step.

Reply:
i reviewed this before seeing #74. I am happy to make a formal policy of using `$`.